build.gradle

```diff
@@ -1,16 +1,18 @@
 buildscript {
     ext.kotlin_version = "1.4.0"
-    ext.python_core_version = "201.6668.115"
-    ext.toml_version = "0.2.120.37-193"
+    ext.python_core_version = "202.6397.124"
+    ext.toml_version = "0.2.128.3278-202"
```

Comment:
We may change the version in this line too.
https://github.com/koxudaxi/poetry-pycharm-plugin/blob/d429afd33013170f58cba0b5b052001e8e04f721/resources/META-INF/plugin.xml#L137

Reply:
Done! I modified that file and also added some configurations to `build.gradle` that will patch the file automatically at build time in the future:
```
patchPluginXml{
    sinceBuild "201.*"
    untilBuild "202.*"
}
```

Reply:
Great!! 
I will merge this PR when complete CI.